lib/index.js

```diff
@@ -124,10 +124,13 @@ exports.getDefaultToESM = async (dirname) => {
 
 exports.getDefaultExport = (exports_, path, type) => {
 
-    if ((type === 'esm' || Path.extname(path) === '.ts') &&
-        exports_ &&
-        typeof exports_ === 'object' &&
-        'default' in exports_) {
+    const exportsObjectWithDefault = exports_ && typeof exports_ === 'object' && 'default' in exports_;
+
+    if ((type === 'esm' || Path.extname(path) === '.ts') && exportsObjectWithDefault) {
+        return exports_.default;
+    }
+
+    if (type === 'cjs' && exportsObjectWithDefault && require(path).__esModule) {
```

Comment:
Rather than re-requiring it, will this do the trick?
```suggestion
    if (type === 'cjs' && exportsObjectWithDefault && exports_.__esModule) {
```

Reply:
it does indeed! PR updated to access property from exports_ vs re-requiring it